packages/experimental/Avatar/MIGRATION.md

```diff
@@ -0,0 +1,158 @@
+# Avatar Migration
+
+## Migration from PersonaCoin component
+
+### Component renames
+
+| `PersonaCoin`                                    | `Avatar`                                                     |
+| ------------------------------------------------ | ------------------------------------------------------------ |
+| `<Avatar size={24} initials="SN" />`             | `<PersonaCoin size='size24' initials="SN" />`                |
```

Comment:
Looks like it's not flipped. I think here it's flipped because width is too small. Here it looks fine: 
![image](https://user-images.githubusercontent.com/11574680/171389961-274d66c2-b0f9-48f1-9a11-2423f315b161.png)

Reply:
oh, I haven't noticed it :) thank you. Fixed